README.md

```diff
@@ -0,0 +1,53 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> This is a project for microverse 2.0 training
+
+![screenshot](./app_screenshot.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo
+
+[Live Demo Link](https://test.com)
+
+
+
+To get a local copy up and running follow these simple example steps.
```

Comment:
- [ ] I'm afraid that you forgot to include the instructions for cloning and setting up the repo locally. Please write descriptive step-by-step instructions for non-technicals to help set up the repo on their machine.
We always need a reminder sometimes. Kindly refer to this [template](https://github.com/microverseinc/readme-template)